content/research/structure-based/pharmacophores.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pharmacophore modeling
+date: 2020-03-03
+publishdate: 2020-03-03
+weight: 10
+nav: false
+---
+
+3D pharmacophore modeling is a powerful tool to encode a ligand`s physicochemical properties in space (pharmacophore features), which are supposed to be important for ligand-target binding. Such a model can be used for virtual screening for novel ligands matching the pharmacophore features. Depending on the available data for a target under investigation, 3D pharmacophores can be generated from either a set of known ligands (*ligand-based*), ligand-macromolecule complexes (*structure/interaction-based*), or macromolecules (apo structures) without binding ligands (*target-based*).
+
+<!--more-->
+
+Pharmacophores can be built upon static structures from e.g. X-ray experiments (*static pharmacophores*), however since ligands and macromolecules are of dynamic nature, molecular dynamic simulations have been more and more employed to add a structure's dynamic behaviour to pharmacophore modeling (*dynamic pharmacophores*). We work on two tools, `T²F-Pharm` and `T²F-Flex`, for static and dynamic target-based pharmacophore modeling, respectively.
```

Comment:
```suggestion
Pharmacophores can be built upon static structures from e.g. X-ray experiments (*static pharmacophores*), however since ligands and macromolecules are of dynamic nature, molecular dynamic simulations have been more and more employed to add a structure's dynamic behaviour to pharmacophore modeling (*dynamic pharmacophores*). We work on two tools, *T²F-Pharm* and *T²F-Flex*, for static and dynamic target-based pharmacophore modeling, respectively.
```
In other projects like *Ratar* and *KiSSim*, the names were italic, not monospaced.